hooks/useRailgunProvider.tsx

```diff
@@ -23,5 +47,5 @@ export const useRailgunProvider = () => {
     setProviderForNetwork(network.railgunNetworkName, provider);
   }, [provider, network.railgunNetworkName]);
 
-  return { isProviderLoaded };
+  return { isProviderLoaded, shieldFees };
```

Comment:
nit: some slight ambiguity since "shield" is a verb everywhere else

```suggestion
  return { isProviderLoaded, shieldingFees };
```

Reply:
Renamed in f7eb5d2